UI/MVC.js

```diff
@@ -0,0 +1,327 @@
+const users = [{ login: 'janny_9991', password: '123456' }, { login: 'katya', password: 'katya1' }, { login: '2', password: '2' }];
```

Comment:
All the staff of controller should be wrapped into IIFE as it currently goes to the global object and that's not good